src/tests/commentsCount.test.js

```diff
@@ -0,0 +1,29 @@
+import countComments from './countcomment.js';
```

Comment:
- [ ] Hi,  The purpose of the test is to test the actual functions that your production code uses currently you are testing a function that has been created in the tests folder. You need to test the function that you are actually using in the modules folder that your website actually uses. Please kindly fix this issue.

Reply:
although the function that I have used inside a separate js file is the same we used in the src directory, I'll change the path, thanks.